shell/client/shell.html

```diff
@@ -1151,6 +1151,91 @@ <h3 id="app-stats">Apps</h3>
   </div>
 </template>
 
+<template name="referrals">
+  {{#if quotaEnabled}}
+  {{setDocumentTitle}}
+  {{>title "About Sandstorm"}}
+  <div class="about">
+    {{#if isPaid}}
+    <h1>Get up to 30GB of free space when your friends share on Oasis!</h1>
+    {{else}}
+    <h1>Get unlimited grains &amp; up to 2GB of free space when
+      your friends share on Oasis!</h1>
+    {{/if}}
+
+    <h2>Your reward</h2>
+
+    {{#if isPaid}}
+    <p>
+      Once someone clicks a sharing link that your friend created,
+      we'll reward you with 2 GB of free storage. You can earn <strong>
+      up to 30GB of free storage</strong> this way!
+    </p>
+    {{else}}
+    <p>
+      Once someone clicks a sharing link that your friend created,
+      we'll reward you with <strong>unlimited grains</strong> and 50
+      MB of free storage.
+      Tell them to share an Etherpad grain or a
+      Game of Y game with you.
+    </p>
+
+    <p>You'll also earn 50 MB per person you successfully
+      refer after the first, <strong>up to 2 GB of free storage.</strong></p>
+    {{/if}}
+
+    <h2>Sharing is referring</h2>
+
+    <p>The best way to refer a friend is to <a href="/grains">open any
```

Comment:
prefer the `{{pathFor grains}}` syntax for internal links, so if we ever rename routes, we won't break links